Source/Genetic.py

```diff
@@ -15,7 +17,7 @@ def __init__(self, mutRate, mutType, popSize, crossType, cutoff, gridSize):
         self.crossType = crossType
         self.cutoff = cutoff
         self.gridSize = gridSize
-        self.NumberofNumbers = 3
+        self.NumberofNumbers = 0
```

Comment:
Why the change here? nothing will loop properly if this isn't set. We really need to just set this when the file is read or something instead of hard coding it